src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CachedSystemStoreProvider.cs

```diff
@@ -129,53 +129,50 @@ public void Remove(ICertificatePal cert)
                 Monitor.IsEntered(s_recheckStopwatch),
                 "LoadMachineStores assumes a lock(s_recheckStopwatch)");
 
-            IEnumerable<FileInfo> trustedCertFiles;
+            SafeX509StackHandle rootStore = Interop.Crypto.NewX509Stack();
+            Interop.Crypto.CheckValidOpenSslHandle(rootStore);
```

Comment:
Do we always call this after NewX509Stack? If yes then might be worth moving the chuck in the "constructor".

Reply:
Not sure. I'm just moving the code here so I'm not going to change it in this PR. If you think it's worth following up on, please do :)